rfcs/convergence/unified-project-board-process.md

```diff
@@ -0,0 +1,82 @@
+# RFC: Unified project board process
+
+---
+
+@tudorpopams
+
+## Summary
+
+Proposes the processes for using the [FluentUI - Unified](https://github.com/orgs/microsoft/projects/395/views/2) project board.
+
+## Background
+
+There have been multiple efforts into trying to unify the way we track, assign, prioritize and estimate issues across FluentUI React teams. With this RFC we're trying to set a process for using a single unified board for all teams and v-teams.
+
+## Problem statement
+
+We currently have 8 project boards that span across all teams and v-teams. This creates redundancy when sharing statuses and makes it hard to have a common [backlog](#backlog) that we can groom and prioritize. Besides this, having separate boards for each team creates desynchronized [sprint](#sprints) timelines and different types of [estimations](#difficulty).
+
+With this RFC we're hoping to solve most of these issues by creating a process on how we should use the unified board and its "ceremonies".
+
+## Detailed Design or Proposal
+
+Our solution to these problems is to start using a [common, unified project board](https://github.com/orgs/microsoft/projects/395/views/2) across all teams and v-teams with the following processes:
+
+1. All teams will run under the same [sprint](#sprints). We will start with 2 weeks long sprints.
+   - Planning and other sprint related ceremonies (ie retrospective) are up to each team.
+1. Ideally all issues assigned to a sprint should be [prioritized](#prioritization) and [estimated](#difficulty).
+1. The managers of each team will have weekly meetings to groom the [backlog](#backlog) and prioritize items.
+   - Teams will have full ownership of breaking down big tasks and planning accordingly
+1. Each team will set up [milestones](#milestones) which will reflect their 4 sprints long road map.
+1. Teams should try to use [labels](https://docs.github.com/en/issues/using-labels-and-milestones-to-track-work/managing-labels) instead of creating new fields.
```

Comment:
should this RFC create and propse separate labels for each feature crew to use ?

Reply:
I think labels might be more team specific, so I'd leave that to each crew to decide.